src/ToggleAsAClass.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+class ToggleAsAClass extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = { toggle: props.initialValue };
```

Comment:
This is has the same effect as `const [toggle, setToggle] = useState(initialValue);`